jupyter-book/chromatin_accessibility/introduction.ipynb

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
<p>Don't forget to add the authors section to also give yourself some credit :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/single-cell-best-practices/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>